get_deps.sh

```diff
@@ -47,10 +47,17 @@ else
 	fi
 fi
 
-git submodule update --init --recursive || true
+#git submodule update --init --recursive || true
```

Comment:
now that it is gone, where it should be called?

Reply:
It shouldn't be. This is needed only for the content of the if statements. I'd argue if you clone without recursive, it needs to run, hence this change.